agenta-web/src/components/EvaluationTable/ABTestingEvaluationTable.tsx

```diff
@@ -203,6 +234,15 @@ const ABTestingEvaluationTable: React.FC<EvaluationTableProps> = ({
         })
     }
 
+    const handleExportClick = () => {
+        const csvData = convertToCsv(
+            rowData,
+            columnDefs.map((col) => col.field),
+        )
+        const filename = `${evaluation.evaluationType}.csv`
```

Comment:
would be nice if filename follow this convention: `<app_name>_<variant_name>_<evaluation_type>.csv`